codespell_lib/data/dictionary.txt

```diff
@@ -2344,7 +2344,7 @@ inappropiate->inappropriate
 inaugures->inaugurates
 inbalance->imbalance
 inbalanced->imbalanced
-inbetween->between
+inbetween->in between
```

Comment:
I'm not sure about this one. I've seen disagreements on dictionaries about it in the past. grammarly.com says it's incorrect, but it does appear on some dictionaries as "informally accepted".